.travis.yml

```diff
@@ -1,10 +1,14 @@
+dist: bionic  # Ubuntu 18.04
```

Comment:
it was not necessary but I believe that is good to used recent then archaic OS version since some old distribution even does not have last versions of some libraries...